tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -92,11 +99,12 @@ static void execute(String... args) throws Exception {
             props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, namespace.getString("bootstrap_server"));
             admin = Admin.create(props);
 
+            boolean readable = Optional.of(namespace.getBoolean("human_readable")).orElse(false);
```

Comment:
we only use this for --describe and replication, please move inside `handleDescribeReplication()`